app/views/images/index.html.erb

```diff
@@ -3,10 +3,19 @@
 <div id="images">
   <% if @images.any? %>
     <% @images.each do |image| %>
+
+      <%= image.name %>
+      <%= form_tag("/add_to_cart", method: "post") do %>
+        <%= hidden_field_tag(:image_id, image.id) %>
+        <%= hidden_field_tag(:image_price, image.price) %>
+        <%= submit_tag("Add to cart") %>
+      <% end %>
```

Comment:
This form should be placed inside the div below. That div is meant to be a wrapper for everything related to the specific image we are displaying atm. That's why we gave it an class of  `"image-<%=image.id%>"`